index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<!-- my web -->
+<body>
+  <main id="main">
+
+    <h1 id="title">Dr. Norman Borlaug</h1> 
+    
+    <div id="img-div"> <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+    <div id="img-caption">"Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists" </div>
+    </div>
+  
+  <div id="tribute-info">
```

Comment:
- [ ] Thank you for providing a clear explanation of the reasoning behind your suggestion. However, I had some trouble understanding the part where you mentioned _**changing the 'div' tag for a different one**_. Could you please clarify which tag you had in mind, and perhaps provide some additional context to make the suggestion easier to follow? 🙏

- [ ] **[OPTIONAL]** You can consider using the add code GitHub feature when you write "div" to make it look like this `div`. This will help distinguish your tags from regular text and make it easier for others to understand. Hope this helps! 👌

  ![image](https://user-images.githubusercontent.com/107952724/235542357-a23ad45d-fa8b-4981-969b-a7a3470c2faf.png)

Reply:
Hi @taldr27, thank you again for your time. Sorry i don't know why, but when i wrote the name of the tag i did it with "less than" and "greater than" simbol, and github just removed them from my comment making the explanation lose meaning. I'm going to solve it, thank you!